packages/nx-plugin/src/executors/cli/executor.ts

```diff
@@ -36,21 +42,30 @@ export default function runAutorunExecutor(
     logger.warn(`DryRun execution of: ${commandString}`);
   } else {
     try {
-      // @TODO use executeProcess instead of execSync -> non blocking, logs #761
-      // eslint-disable-next-line n/no-sync
-      execSync(commandString, commandStringOptions);
+      await executeProcess({
+        ...createCliCommandObject({ command, args: cliArgumentObject }),
+        observer: {
+          onError: error => {
+            logger.error(error.message);
+          },
+          onStdout: data => {
+            process.stdout.write(data);
+          },
```

Comment:
You can avoid the extra callbacks here:

```suggestion
          onError: logger.error,
          onStdout: process.stdout.write,
```

This'll reduce your LOC per function under 50, so you don't need to disable the lint rule. :slightly_smiling_face: